pkg/web/static/js/userProfile.js

```diff
@@ -0,0 +1,53 @@
+// Helper function to add the same event handler for multiple events
+const addEventListeners = (el, evts, fn) => {
+  evts.split(' ').forEach(e => el.addEventListener(e, fn, false));
+}
+
+// Elements referenced by this script
+const changePwdModal = document.getElementById('change_pwd_modal');
+const changePwdForm = document.getElementById('change-pwd-form')
+const closePwdModalBtn = document.getElementById('close-pwd-mdl-btn');
+const passwordInput = document.getElementById('password');
+const confirmPassword = document.getElementById('confirm-password');
+const changePasswordBtn = document.getElementById('change-password-btn');
+
+// Reset the form when change password modal is closed in case user entered some values without submitting.
+closePwdModalBtn.addEventListener('click', () => {
+  changePwdForm.reset();
+});
+
+// Enable the change password button when the password and confirm password inputs are identical.
+// TODO: should we add any other visual indication that the passwords do not match?
+const checkPasswords = e => {
+  if (passwordInput.value == confirmPassword.value && passwordInput.value != '') {
```

Comment:
Should this be `===` and `!==`? I can never remember with Javascript.

Reply:
`===` and `!==` or `strict equality` should be used. It will ensure the types also match.

Reply:
ok, so only use `==` and `!=` when it can be undefined or `null`?

Reply:
Yes. Even then, you'll still often see `===` or `!==` used.